dashboard/app/assets/javascripts/components/share_dialog.jsx

```diff
@@ -21,66 +23,36 @@ window.dashboard.ShareDialog = (function (React) {
     },
 
     getInitialState: function () {
-      return {
-        hidden: false
-      };
+      return { isOpen: true };
     },
 
     componentWillReceiveProps: function (newProps) {
-      this.setState({hidden: false});
+      this.setState({isOpen: true});
     },
 
-    componentDidMount: function () {
-      this.refs.dialog.getDOMNode().focus();
-    },
-
-    componentDidUpdate: function () {
-      if (!this.state.hidden) {
-        this.refs.dialog.getDOMNode().focus();
-      }
-    },
-
-    onKeyDown: function (event) {
-      if (event.key === 'Escape') {
-        this.closeDialog();
-      }
-    },
-
-    closeDialog: function () {
-      this.setState({hidden: true});
+    close: function () {
+      this.setState({isOpen: false});
     },
 
     render: function () {
-      if (this.state.hidden) {
-        return <div/>;
-      }
-
       // TODO - Can we now make SendToPhone completely Reactified?
```

Comment:
I would say yes in this PR. It should be removed in part 3 (https://github.com/code-dot-org/code-dot-org/pull/6171), though it doesn't look like it is there either.